app/views/prison/visits/_visitor_details.html.erb

```diff
@@ -5,14 +5,14 @@
 
 <% if Nomis::Feature.contact_list_enabled?(@visit.prison_name) && !@visit.contact_list_unknown? %>
```

Comment:
This has been changed in https://github.com/ministryofjustice/prison-visits-2/pull/864 which needs to be merged before release